runtime/near-vm-logic/src/config.rs

```diff
@@ -187,6 +187,27 @@ impl Default for VMLimitConfig {
 
 #[derive(Debug, Serialize, Deserialize, Clone, Hash, PartialEq, Eq)]
 pub struct ExtCostsConfig {
+    // Base cost for multiexp
+    pub alt_bn128_g1_multiexp_base: Gas,
+
+    // byte cost for multiexp
+    pub alt_bn128_g1_multiexp_byte: Gas,
```

Comment:
This is an issue, since the old protocol doesn't have such fields. 
We haven't completed a config upgrade that introduces new fields, so I currently don't know what's the best way to introduce them.

But, check out how we want to add such fields for EVM branch: https://github.com/near/nearcore/pull/3257
We basically added a new config and made it skipped form genesis, but deserialized from default.